Makarava/TM_BulkThumbnailCreator/TM_BulkThumbnailCreator/Constant.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Threading;
+
+namespace TM_BulkThumbnailCreator
+{
+    static public class Constant
+    {
+        public static string mainMenu = "\tPlease select an action to make with your pictures: " +
+                "\n\t1.Size changing" +
+                "\n\t2.Rename all images";
+
+        public static string userMessageForRename = "\tPlease type common name for all pictures:";
+
+        public static string userMessageForResize = "\tResizing process is started";
+
+        public static string notificationToReturnToMainMenu = "\tYour process is done. Type anything to return to main menu";
+
+        public static string cancellationMessage = "\tYour process is done. Type anything to return to main menu";
+
+        public static void WelcomingMessage(string initialFolder, string folderAfterChanges)
```

Comment:
If this's utility class for constants/messages why did you add logics/methods hire? remove usage of Console class outside the class